dapr/proto/scheduler/v1/scheduler.proto

```diff
@@ -0,0 +1,158 @@
+syntax = "proto3";
+
+package dapr.proto.scheduler.v1;
+
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/dapr/dapr/pkg/proto/scheduler/v1;scheduler";
+
+service Scheduler {
+	// ScheduleJob is used by the daprd sidecar to schedule a job.
+	rpc ScheduleJob(ScheduleJobRequest) returns (ScheduleJobResponse) {}
+	// Get a job
+	rpc GetJob(GetJobRequest) returns (GetJobResponse) {}
+	// DeleteJob is used by the daprd sidecar to delete a job.
+	rpc DeleteJob(DeleteJobRequest) returns (DeleteJobResponse) {}
+	// WatchJobs is used by the daprd sidecar to connect to the Scheduler
+	// service to watch for jobs triggering back.
+	rpc WatchJobs(stream WatchJobsRequest) returns (stream WatchJobsResponse) {}
+}
+
+message Job {
+  // The schedule for the job.
+  optional string schedule = 1;
+
+  // Optional: jobs with fixed repeat counts (accounting for Actor Reminders).
+  optional uint32 repeats = 2;
+
+  // Optional: sets time at which or time interval before the callback is invoked for the first time.
+  optional string due_time = 3;
```

Comment:
Should probably be a google.protobuf.Timestamp?

Reply:
The type is for consistency with actor reminders due time, which is string.